aboutcitus/what_is_citus.rst

```diff
@@ -10,7 +10,16 @@ Citus extends the underlying database rather than forking it, which gives develo
 When to Use Citus
 -----------------
 
-There are two situations where Citus particularly excels: real-time analytics and multi-tenant applications.
+Citus serves many use cases. Two common ones are scaling multi-tenant (B2B) databases and real-time analytics.
+
+Multi-Tenant Database
+~~~~~~~~~~~~~~~~~~~~~
+
+Most B2B applications already have the notion of a tenant, customer, or account built into their data model. In this model, the database serves many tenants, each of whose data is separate from other tenants.
+
+Citus provides full SQL coverage for this workload, and enables scaling out your relational database to 100K+ tenants. Citus also adds new features for multi-tenancy. For example, Citus supports tenant isolation to provide performance guarantees for large tenants, and has the concept of reference tables to reduce data duplication across tenants.
+
+These capabilities allow you to scale out your tenants data across many machines, and easily add more CPU, memory, and disk resources. Further, sharing the same database schema across multiple tenants makes efficient use of hardware resources and simplifies database management.
```

Comment:
Also there's an inconsistent voice in these paragraphs, where some sentences talk directly to the reader ("Citus allows you to...") while others talk more generally about the properties of the system ("Citus supports tenant isolation...").

It's not a huge deal, just something I try to watch out for in my own writing.